### PR TITLE
feat: session state filter dropdown

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
Turn the session state filter into a dropdown to save space.

New design:
image

## Linear Ticket
https://linear.app/autokitteh/issue/UI-775/change-the-sessions-filter-to-dropdown

## What type of PR is this? (check all applicable)

-   [x] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/organisms/deployments/sessions/table.tsx [hunk 1]
```diff
@@ -196,34 +196,29 @@ export const SessionsTable = () => {
...
 						<Typography className="text-base" element="h2">
 							{t("tableTitle")}
 						</Typography>
-						<div className="flex flex-wrap items-center justify-between gap-2.5">
-							<SessionsTableFilter onChange={handleFilterSessions} sessionStats={sessionStats} />
-							<RefreshButton isLoading={isLoading} onRefresh={() => refreshData()} />
-						</div>
+						<div className="w-full" />
+						<SessionsTableFilter onChange={handleFilterSessions} sessionStats={sessionStats} />
+						<RefreshButton isLoading={isLoading} onRefresh={() => refreshData()} />
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/organisms/deployments/sessions/table.tsx:205]: refactored

File: src/components/organisms/deployments/sessions/table.tsx
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/organisms/deployments/sessions/table.tsx:1]: refactored

File: src/components/organisms/deployments/sessions/filter.tsx [hunk 2]
```diff
@@ -54,38 +72,51 @@ export const SessionsTableFilter = ({ onChange, sessionStats }: SessionTableFilt
...
+						</Button>
 
-			<Button
-				className={buttonClass(SessionStateType.stopped)}
-				onClick={() => handleButtonClick(SessionStateType.stopped)}
-			>
-				{t("stopped")} ({sessionCounts.stopped})
-			</Button>
+						<Button
+							className={buttonClass(SessionStateType.running)}
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/organisms/deployments/sessions/filter.tsx:84]: done

File: src/components/organisms/deployments/sessions/filter.tsx [hunk 3]
```diff
@@ -54,38 +76,51 @@ export const SessionsTableFilter = ({ onChange, sessionStats }: SessionTableFilt
...
+							onClick={() => handleButtonClick(SessionStateType.completed)}
+						>
+							{t("completed")} ({sessionCounts.completed})
+						</Button>
+					</div>
+				}
 			>
-				{t("completed")} ({sessionCounts.completed})
-			</Button>
+				<div className="flex">
```
Comment (J1za) [inline src/components/organisms/deployments/sessions/filter.tsx:117]: remove parent div

Comment (RonenMars) [reply src/components/organisms/deployments/sessions/filter.tsx:117]: refactored

**Review by J1za — APPROVED** [review]

**Pull request merged**